vm.go

```diff
@@ -758,6 +758,7 @@ func init() {
 				oci.setTop(ofn + (e.l.top - nfn)) // correct top
 				oci.frame = e.l.stack[base:oci.top]
 				oci.savedPC = nci.savedPC
+				oci.code = nci.code               // correct code (savedPC indexes nci->code)
```

Comment:
I'd do these in a tuple assignment, so they're not accidentally separated. I.e.:
```go
oci.savedPC, oci.code = nci.savedPC, nci.code // savedPC indexes code
```